org.eclipse.jdt.ui.tests.refactoring/test cases/org/eclipse/jdt/ui/tests/refactoring/ExtractMethodTestSetup.java

```diff
@@ -169,5 +169,6 @@ public IPackageFragment getVarargsPackage() {
 	public IPackageFragment getFieldInitializerPackage() {
 		return fFieldInitializerPackage;
 	}
+
```

Comment:
Don't commit changes which are only formatting.

Reply:
That is noted with thanks and will address it.